From 6e875bbb8eca61a567c120a3d21ca6dba74b76b6 Mon Sep 17 00:00:00 2001
From: Ziyuan Xu <xzy.xu@rock-chips.com>
Date: Sun, 6 Sep 2020 19:34:31 +0800
Subject: [PATCH] ARM: configs: rv1126-tb.config: enable ov2718

Till now, the paired cameras for tb product:
- ov2718   1920x1080
- os04a10  2688x1520

Signed-off-by: Ziyuan Xu <xzy.xu@rock-chips.com>
Change-Id: I431aafc5168a56c1d84bd4d11775b3edc9016ae4
---
 arch/arm/configs/rv1126-tb.config | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/configs/rv1126-tb.config b/arch/arm/configs/rv1126-tb.config
index d00c782a98de..729df48a8cb0 100644
--- a/arch/arm/configs/rv1126-tb.config
+++ b/arch/arm/configs/rv1126-tb.config
@@ -66,6 +66,8 @@ CONFIG_USB_XHCI_HCD=m
 CONFIG_USB_XHCI_PLATFORM=m
 # CONFIG_VIDEO_GC4C33 is not set
 # CONFIG_VIDEO_IMX347 is not set
+# CONFIG_VIDEO_IMX378 is not set
+CONFIG_VIDEO_OV2718=y
 # CONFIG_VIDEO_OV4689 is not set
 CONFIG_WL_ROCKCHIP=m
 CONFIG_INITCALL_ASYNC=y
-- 
2.35.3

